github-dark.css

```diff
@@ -415,7 +415,7 @@
   .inline-form .btn-plain, .pr-toolbar, .diffbar, .shade-gradient,
   #categories .list, .gist-banner, .select-menu-item-parent, .main-content,
   .showcase-page-header, .showcase-page-pattern:after, .description-inline,
-  .result-group a, .border-wrap, .select-menu-item, a.feed-icon {
+  .result-group a, .border-wrap, .select-menu-item, a.feed-icon, session-current {
```

Comment:
@silverwind https://github.com/StylishThemes/GitHub-Dark/pull/358/commits/c9aab5ee221ecbd2359028e31be315a9648f530f

still needs fixing though, idk why but I cant do it